unblob/logging.py

```diff
@@ -66,6 +66,31 @@ def add_pid_to_log_message(
     return event_dict
 
 
+def move_exception_to_exc_info(
+    _logger, _method_name: str, event_dict: structlog.types.EventDict
+):
+    """Workaround for a structlog's shortcoming.
+
+    ConsoleRenderer has "special" requirements for keys, e.g. raises an Exception if there
+    is an "exception" key in the passed in event_dict, and its value is an Exception instance.
+
+    Counterintuitively ConsoleReader requires the exception's value to be a string!
+
+    We have some code that builds a dict for logging that includes this "exception" key:
+    e.g. logging report.UnknownError and its derivatives.
+
+    This is necessary a hackish solution, that depends on structlog.processors.format_exc_info
+    to do its work after it.
+    """
+    try:
+        exc = event_dict["exception"]
+        if isinstance(exc, Exception):
+            event_dict["exc_info"] = exc
+    except KeyError:
+        pass
+    return event_dict
```

Comment:
This eventually duplicates the value stored under the `exception` to the `exc_info`.
I would *move* it rather then *copy* it. 
e.g. in this case, I think we don't need the original `exception` key (let structlog handle that).

Reply:
The functionality is there, as `structlog.processors.format_exc_info` will replace the `exception` key if `exc_info` is present.

You are right however, that by moving it it becomes more evident - will move it, this is the proper solution - thanks!

Reply:
workaround is different: model change